package.json

```diff
@@ -11,9 +11,6 @@
     "format": "npx pretty-quick",
     "lint": "npx eslint src/*.js --fix"
```

Comment:
## Support for **JSX** files.
Add a Regex selector to check for linting issues in both js and jsx.

Edit the script to match this:
"lint": "npx eslint src/*.{js,jsx} --fix"
@samuel-shyaka-dus